src/UserFacingError.hack

```diff
@@ -20,22 +16,22 @@ class UserFacingError extends \Exception {
         'message' => string,
         ?'location' => shape('line' => int, 'column' => int),
         ?'path' => vec<arraykey>,
+        ?'cause' => shape(
```

Comment:
It doesn't really matter much, but you could put this inside `extensions` to more closely follow the spec: https://spec.graphql.org/draft/#sel-GAPHRPZCAACCBx6b

Reply:
Nice, updated.